default_organisation.bash

```diff
@@ -212,6 +213,25 @@ cmd_recrypt() {
 	git_commit "Recrypt $* for ${target_groups[*]}."
 }
 
+# Check for an extension update
+cmd_update() {
+	echo "Looking for update..."
+	tmp_file="/tmp/pass-group-$EXTENSION_NAME.bash"
```

Comment:
This is not portable.

`pass` implement a `tmpdir` function but we do not want to use it as it makes sure to make a secure temporary file and fail if `/dev/shm` does not work.

Have a look at `mktemp`:
```bash
local tmp_file="$(mktemp).bash"
```

Reply:
Fixed